recipes/jasper/all/test_v1_package/test_package.c

```diff
@@ -0,0 +1,43 @@
+#include <jasper/jasper.h>
```

Comment:
Does this file need to be duplicated or should it be?

Reply:
duplicate how? Sorry, I didn't get your point ...

Reply:
Ah, do you mean duplicated file in test_v1_package folder. Yes, let me remove it. Thanks!

Reply:
Done, it has been removed. Thanks!